From 95bef19aae4563d8bceb2e3e91d396c4e4473073 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 25 Jun 2024 17:27:22 +0200
Subject: [PATCH 0936/1695] dts: stm32: reduce access-controllers-cells to 1 on
 stm32mp21

Limit the number of access-controllers cells to 1

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 0b04e39ca6ee89832e2c46bf2d1d96d374eb0e7c dts: stm32: reduce access-controllers-cells to 1 on stm32mp21]
Change-Id: I0afca7ad44f579cc5c7c4b961fbea3eeab9350e0
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/389579
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 32 +++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 9db47c5e4..8ae768b32 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -219,7 +219,7 @@
 			reg = <0x42080000 0x1000>;
 			#address-cells = <1>;
 			#size-cells = <1>;
-			#access-controller-cells = <2>;
+			#access-controller-cells = <1>;
 
 			lptimer1: timer@40090000 {
 				compatible = "st,stm32mp21-lptimer";
@@ -229,7 +229,7 @@
 				clock-names = "mux";
 				#address-cells = <1>;
 				#size-cells = <0>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM1_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_LPTIM1_ID>;
 				status = "disabled";
 
 				counter {
@@ -260,7 +260,7 @@
 				reg = <0x400e0000 0x400>;
 				interrupts = <GIC_SPI 115 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_USART2>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_USART2_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_USART2_ID>;
 				status = "disabled";
 			};
 
@@ -269,7 +269,7 @@
 				reg = <0x400f0000 0x400>;
 				interrupts = <GIC_SPI 116 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_USART3>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_USART3_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_USART3_ID>;
 				status = "disabled";
 			};
 
@@ -278,7 +278,7 @@
 				reg = <0x40100000 0x400>;
 				interrupts = <GIC_SPI 126 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_UART4>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_UART4_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_UART4_ID>;
 				status = "disabled";
 			};
 
@@ -287,7 +287,7 @@
 				reg = <0x40110000 0x400>;
 				interrupts = <GIC_SPI 127 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_UART5>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_UART5_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_UART5_ID>;
 				status = "disabled";
 			};
 
@@ -299,7 +299,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				i2c-analog-filter;
-				access-controllers = <&rifsc STM32MP21_RIFSC_I2C1_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_I2C1_ID>;
 				status = "disabled";
 			};
 
@@ -311,7 +311,7 @@
 				#address-cells = <1>;
 				#size-cells = <0>;
 				i2c-analog-filter;
-				access-controllers = <&rifsc STM32MP21_RIFSC_I2C2_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_I2C2_ID>;
 				status = "disabled";
 			};
 
@@ -320,7 +320,7 @@
 				reg = <0x40220000 0x400>;
 				interrupts = <GIC_SPI 136 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_USART6>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_USART6_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_USART6_ID>;
 				status = "disabled";
 			};
 
@@ -329,7 +329,7 @@
 				reg = <0x40330000 0x400>;
 				interrupts = <GIC_SPI 114 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_USART1>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_USART1_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_USART1_ID>;
 				status = "disabled";
 			};
 
@@ -338,7 +338,7 @@
 				reg = <0x40370000 0x400>;
 				interrupts = <GIC_SPI 148 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_KER_UART7>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_UART7_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_UART7_ID>;
 				status = "disabled";
 			};
 
@@ -348,7 +348,7 @@
 				interrupts = <GIC_SPI 145 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_BUS_HASH2>;
 				resets = <&rcc HASH2_R>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_HASH2_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_HASH2_ID>;
 				status = "disabled";
 			};
 
@@ -357,7 +357,7 @@
 				reg = <0x42020000 0x400>;
 				clocks = <&clk_rcbsec>, <&rcc CK_BUS_RNG2>;
 				clock-names = "rng_clk", "rng_hclk";
-				access-controllers = <&rifsc STM32MP21_RIFSC_RNG2_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_RNG2_ID>;
 				resets = <&rcc RNG2_R>;
 			};
 
@@ -367,7 +367,7 @@
 				interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_BUS_IWDG1>, <&rcc LSI_CK>;
 				clock-names = "pclk", "lsi";
-				access-controllers = <&rifsc STM32MP21_RIFSC_IWDG1_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_IWDG1_ID>;
 				status = "disabled";
 			};
 
@@ -377,7 +377,7 @@
 				interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&rcc CK_BUS_IWDG2>, <&rcc LSI_CK>;
 				clock-names = "pclk", "lsi";
-				access-controllers = <&rifsc STM32MP21_RIFSC_IWDG2_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_IWDG2_ID>;
 				status = "disabled";
 			};
 
@@ -389,7 +389,7 @@
 				i2c-analog-filter;
 				#address-cells = <1>;
 				#size-cells = <0>;
-				access-controllers = <&rifsc STM32MP21_RIFSC_I2C3_ID 0>;
+				access-controllers = <&rifsc STM32MP21_RIFSC_I2C3_ID>;
 				status = "disabled";
 			};
 		};
-- 
2.47.2

